products/db-checkbook/sql_query/query.sql

```diff
@@ -1,3 +1,303 @@
+UPDATE capital_projects SET col_category = 'Fixed Asset'
```

Comment:
sql is a more apt name for this folder, and the file itself could use a more descriptive name